src/buffer/out/cursor.h

```diff
@@ -76,7 +76,7 @@ class Cursor final
 
     void CopyProperties(const Cursor& OtherCursor) noexcept;
 
-    void DelayEOLWrap(const til::point coordDelayedAt) noexcept;
+    void DelayEOLWrap() noexcept;

```

Comment:
I love this. Removing parameters that should be known by internal state. :smile: